css/css-grid/abspos/descendant-static-position-001-ref.html

```diff
@@ -33,35 +33,35 @@
 There should be no red:
 
 <div class="grid">
-    <div class="absolute" style="margin-top:2px">
+    <div class="absolute">
       <div class="content"></div>
       <div class="content"></div>
     </div>
 </div>
```

Comment:
So the rendering that we're expecting from this reference case seems a bit odd, since we'll now be using a static **position** determined from the grid's content-box, but we're still using a **size** determined from the grid area.  (This is kind of implicit, from the fact that we're expecting these `content` elements to end up on the same line, i.e. we're letting the abspos resolve to be wider than the grid's content box, even though that's now what we're using to do our static positioning.)

That feels a little weird, and it means we're not really ignoring the grid positioning properties after all for statically positioned grid children that use the grid for their containing block (as https://github.com/w3c/csswg-drafts/issues/7661 seemed to be indicating that we wanted to do).

I wonder if the behavior we're ending up with here (asserted by this reference case) is actually what we want?  Possibly we need a bit more spec clarification on this point and might want a different outcome?  I'll post a comment on the PR with some more thoughts and with a jsfiddle that gets a bit more clearly at the strangeness, since the textbox here feels a bit limiting for links/screenshots/etc.